validmind/documentation
# Remove rendered site output from repo

## Internal Notes for Reviewers

This PR 

- removes the `site/_site/` folder with the rendered docs site from the repo; AND
- adds the folder to the `.gitignore` file to prevent it from being re-added

These changes add the following highlight from our docs CI/CD deployment proposal. The output is no longer needed in the repo as it will now be rendered on the fly when needed for PR previews or staging and prod sites. 

<img width="1532" alt="image" src="https://github.com/user-attachments/assets/57300273-b6fd-4e73-9bb6-8f71019c7216">

## Proof it works 

Run `quarto render` and enter `git status` (`gst` alias) and there should be no output files to commit:

<img width="882" alt="image" src="https://github.com/user-attachments/assets/0ce6af81-fb27-4b70-9cf5-b7b95589cccb">

## How to test

Check out this PR locally, run `quarto render` and then `git status`. There should be no new or modified files shown. 



## External Release Notes
